droneapi/lib/__init__.py

```diff
@@ -173,17 +173,20 @@ class Battery(object):
     """
     Battery info available
 
-    :param battery_voltage: Battery voltage in millivolts
-    :param battery_current: Battery current, in 10 * milliamperes, -1: autopilot does not measure the current
-    :param battery_level: Remaining battery energy, -1: autopilot estimate the remaining battery
+    :param voltage: Battery voltage in millivolts
+    :param current: Battery current, in 10 * milliamperes, -1: autopilot does not measure the current
+    :param level: Remaining battery energy, -1: autopilot estimate the remaining battery
```

Comment:
The changes I plan to make to this section are:

- add full stops :-)
- Now these return ```None``` document that

So:
```
:param voltage: Battery voltage in millivolts.
:param current: Battery current, in 10 * milliamperes. ``None`` if the autopilot does not support current measurement.
:param level: Remaining battery energy. ``None`` if the autopilot cannot estimate the remaining battery.
```

Note the change to description of "level". This is an " educated guess" on the behaviour - the original text (from the specification) does not make any sense.